Formula/hledger.rb

```diff
@@ -1,9 +1,10 @@
 class Hledger < Formula
   desc "Easy plain text accounting with command-line, terminal and web UIs"
   homepage "https://hledger.org/"
-  url "https://hackage.haskell.org/package/hledger-1.26.1/hledger-1.26.1.tar.gz"
-  sha256 "245af2ec616a754fb488a7ad8ffa360732b733d96c7d6b1e49d4b0fbd435e247"
+  url "https://github.com/simonmichael/hledger/archive/refs/tags/1.27.tar.gz"
+  sha256 "1a3d1d321cf5fb9ae6d214a5a4a71775680a8146cf43f77e523a2e6c0af9d366"
```

Comment:
Went with the GitHub tarball to simplify the installation and avoid maintaining our own build files, but haven't looked into any differences with Hackage tarball

Reply:
I believe the Hackage tarballs and github release binaries usually correspond exactly (this should be verifiable by comparing the tarballs with `cabal sdist PKG` made in the git checkout of the hash shown in the binaries' --version).